### PR TITLE
Create chocolate roll

### PR DESCRIPTION
**Pull request by alexandrapurcarea**

### CODE REVIEW

File: desserts/medium/biscuit-salami.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+* 200g butter biscuits (you can add more to taste - Leibniz type biscuits should work best)
+* 50g cocoa powder
+* 130g butter
+* 100ml milk
+* 125g sugar (either brown or white)
+* 50g dark chocolate (at least 50% cocoa)
+* rum essence (not necessary but it makes the taste *much* better - any Polish store should have it)
+* vanilla sugar
+* powdered sugar
+* *extra filling*: nuts, almonds, raisins, cranberries, candied fruit, finely chopped Turkish delight, coconut flakes, or several of these!
```
Comment (elenalape) [inline desserts/medium/biscuit-salami.md]: good shout on the Turkish delight

File: desserts/medium/biscuit-salami.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Method
+
+1. Melt the butter and chocolate on low heat until fully melted. Make sure not to melt the butter more than necessary. Let the mixture cool down.
```
Comment (elenalape) [inline desserts/medium/biscuit-salami.md]: on the hob or microwave is ok?

**Review by elenalape — CHANGES_REQUESTED** [review]

**Comment by elenalape** [comment]
Just as a side note, I think the translated name of the dish is confusing, it doesn't sound like a dessert

File: desserts/medium/biscuit-salami.md (hunk 2 above)
Comment (alexandrapurcarea) [reply desserts/medium/biscuit-salami.md]: Microwave tends to cause disasters for me so I'd recommend the hob, to control melting it just as much as you need.

File: desserts/medium/biscuit-salami.md (hunk 1 above)
Comment (alexandrapurcarea) [reply desserts/medium/biscuit-salami.md]: Yes, that's my favorite filling!

**Comment by alexandrapurcarea** [comment]
> Just as a side note, I think the translated name of the dish is confusing, it doesn't sound like a dessert

Just changed the title!

**Review by neanias — APPROVED** [review]

**Review by qaisjp — APPROVED** [review]

**Pull request merged**